docs/install.md

```diff
@@ -0,0 +1,150 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+
+# Instructions to Build and Install Private Data Objects
+
+Private Data Objects (PDO) can be installed directly to a host system or
+it may be installed through Docker. This document contains instructions
+for both methods of installation.
+
+PDO requires installation of the SDK for Intel Software Guard Extensions
+(SGX). SGX can be used either in simulation mode or hardware
+mode. Simulation mode will work even when there is no processor support
+for SGX. However, simulation mode is not secure and is suitable soley
+for development and demonstration. Hardware mode, which uses trusted
+execution capabilities in the processor to protect confidentiality and
+integrity of computation, requires appropriate processor
+support. Information about supported processors is provided below.
+
+## Recommended Host System
+
+The recommended host-system configuration for Private Data Objects is to
+separate the Private Data Objects components from the Sawtooth components. This
+means (at least) two different physical systems if using SGX-enabled hardware.
+If running in SGX simulation mode, this could be two virtual machines or
+containers.
+
+Sawtooth (and the PDO transaction processors for Sawtooth) should be run on
+Ubuntu 16.04.
+
+Private Data Objects services (specifically the enclave service, provisioning
+service, and the client) should be run on Ubuntu 18.04. PDO has been tested on
+Ubuntu 16.04, 17.10, and 18.04.
+
+Sawtooth and PDO may run on other Linux distributions, but the installation
+process is likely to be more complicated, and the use of other distributions is
+not supported by their respective communities at this time.
+
+## <a name="SGX">Intel Software Guard Extensions (SGX)</a>
+
+Private Data Objects uses the trusted execution capabilities provided by
+Intel SGX to protect integrity and confidentiality. More information
+about SGX can be found on the
+[Intel SGX website](https://software.intel.com/en-us/sgx) including
+[detailed installation instructions](https://download.01.org/intel-sgx/linux-2.4/docs/Intel_SGX_Installation_Guide_Linux_2.4_Open_Source.pdf).
+
```

Comment:
Could  we avoid "including detailed installation instructions." and the link to the doc? This seems to suggest that user must first read the doc, install SGX before proceeding, while this is true. You have very nice two line-instruction further down as to how to install SGX 2.4. 

(The document in my opinion is scary, there is mention of packages to download without links. In hindight, the document will do well to borrow your two line script for installation and putting them on page 1!)

Reply:
agreed that the doc is scary... but we cannot document all of the "special cases". suggest we leave it as is.

Reply:
in that case it might be better to move the link to the documentation after our two-line installation guide for sgx. My concern is if given at this point, the user is made to believe that the user needs to install sgx following this guide. (at least i tried to do that for a while before moving on and seeing what else is there in the build doc)